project_classic/cookiecutter.json

```diff
@@ -0,0 +1,119 @@
+{
+  "title": "Project Title",
+  "description": "A new project using Plone 6.",
+  "project_slug": "{{ cookiecutter.title | slugify }}",
+  "hostname": "{{ cookiecutter.project_slug }}.example.com",
+  "author": "Plone Foundation",
+  "email": "collective@plone.org",
+  "use_prerelease_versions": "{{ 'No' | use_prerelease_versions }}",
+  "plone_version": "{{ cookiecutter.use_prerelease_versions | latest_plone }}",
+  "python_package_name": "{{ cookiecutter.project_slug|replace(' ', '')|replace('-', '.') }}",
+  "language_code": ["en", "de", "es", "pt-br", "nl", "fi"],
+  "github_organization": "collective",
+  "container_registry": ["github", "docker_hub", "gitlab"],
+  "devops_cache": ["1", "0"],
+  "devops_ansible": ["1", "0"],
+  "devops_gha_deploy": ["1", "0"],
+  "__feature_headless": "0",
```

Comment:
This isn't your fault, but I think `feature_headless` is a confusing name (it sounds like there's no frontend, but really we're using it to mean that the Volto frontend is enabled). I want to rename it to `enable_volto_frontend` but I can make a separate issue for that.